File: 1. 단계별로_풀어보기/1. 입출력과 사칙연산/3_1001.js
/**
 * 1-3. 1001. A-B
 *
 * 입력
 *   A, B : 10 이하의 자연수
 *
 * 출력
 *   A - B
 */

const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().split(' ')
let [a, b] = input

a = parseInt(a)
b = parseInt(b)

console.log(a - b)

File: 1. 단계별로_풀어보기/1. 입출력과 사칙연산/3_1001.py
# 1-3. 1001. A-B
#
# 입력
#  A, B: 10 미만의 자연수
#
# 출력
#  A - B

import sys

input = sys.stdin.readline
output = sys.stdout.write

A, B = map(int, input().split())

print(A - B)